src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -499,22 +499,22 @@ public Uri(Uri baseUri, Uri relativeUri)
 
```

Comment:
Shouldn't `relativeUri` be nullable in this ctor?

Reply:
Yup.  Fixed.